compilers/concrete-compiler/compiler/lib/Bindings/Python/CompilerAPIModule.cpp

```diff
@@ -1149,20 +1149,24 @@ void mlir::concretelang::python::populateCompilerAPISubmodule(
           [](::concretelang::clientlib::ClientParameters clientParameters,
              ::concretelang::clientlib::KeySetCache *cache,
              uint64_t secretSeedMsb, uint64_t secretSeedLsb,
-             uint64_t encSeedMsb, uint64_t encSeedLsb) {
+             uint64_t encSeedMsb, uint64_t encSeedLsb,
+             std::map<uint32_t, LweSecretKey> initialLweSecretKeys) {
             SignalGuard signalGuard;
             auto optCache =
                 cache == nullptr
                     ? std::nullopt
                     : std::optional<::concretelang::clientlib::KeySetCache>(
                           *cache);
-            return key_set(clientParameters, optCache, secretSeedMsb,
-                           secretSeedLsb, encSeedMsb, encSeedLsb);
+            return key_set(clientParameters, optCache, initialLweSecretKeys,
+                           secretSeedMsb, secretSeedLsb, encSeedMsb,
+                           encSeedLsb);
           },
           pybind11::arg().none(false), pybind11::arg().none(true),
           pybind11::arg("secretSeedMsb") = 0,
           pybind11::arg("secretSeedLsb") = 0, pybind11::arg("encSeedMsb") = 0,
-          pybind11::arg("encSeedLsb") = 0)
+          pybind11::arg("encSeedLsb") = 0,
+          pybind11::arg("initialLweSecretKeys") =
+              std::map<uint32_t, pybind11::bytes>())
```

Comment:
why it's bytes and not LweSecretKey here?

Reply:
it was initially using bytes, and I didn't update this after the change